src/Chapter1.hs

```diff
@@ -640,8 +662,10 @@ Implement a function that returns the first digit of a given number.
 You need to use recursion in this task. Feel free to return to it later, if you
 aren't ready for this boss yet!
 -}
-
-firstDigit n = error "firstDigit: Not implemented!"
+firstDigit :: Int -> Int
+firstDigit n
+  | div n 10==0 = mod n 10
+  | otherwise = firstDigit (div n 10)
```

Comment:
Yeap this is an absolutely correct idea!
Note, that here you also need to think about the case with negative numbers.

Also, instead of `div n 10 == 0 = mod n 10`, you can do:

```haskell
| absN < 10 = absN
```